app/src/main/java/framgia/com/ichat/data/model/User.java

```diff
@@ -1,19 +1,99 @@
 package framgia.com.ichat.data.model;
 
+import android.net.Uri;
+
+import com.google.firebase.auth.FirebaseUser;
+
+import java.util.Calendar;
+
 public class User {
+    private String mUid;
     private String mEmail;
     private String mPassword;
+    private String mDisplayName;
+    private String mPhotoUrl;
+    private long mLastSignIn;
+    private boolean mStatus;
+
+    public User() {
+    }
 
     public User(String email, String password) {
         mEmail = email;
         mPassword = password;
     }
 
-    public String getmEmail() {
+    public User(String uid, String email, String displayName, String photoUrl, long lastSignIn, boolean status) {
+        mUid = uid;
+        mEmail = email;
+        mDisplayName = displayName;
+        mPhotoUrl = photoUrl;
+        mLastSignIn = lastSignIn;
+        mStatus = status;
+    }
+
+    public String getUid() {
+        return mUid;
+    }
+
+    public void setUid(String uid) {
+        mUid = uid;
+    }
+
+    public String getEmail() {
         return mEmail;
     }
 
-    public String getmPassword() {
+    public void setEmail(String email) {
+        mEmail = email;
+    }
+
+    public String getPassword() {
         return mPassword;
     }
+
+    public String getDisplayName() {
+        return mDisplayName;
+    }
+
+    public void setDisplayName(String displayName) {
+        mDisplayName = displayName;
+    }
+
+    public String getPhotoUrl() {
+        return mPhotoUrl;
+    }
+
+    public void setPhotoUrl(String photoUrl) {
+        mPhotoUrl = photoUrl;
+    }
+
+    public long getLastSignIn() {
+        return mLastSignIn;
+    }
+
+    public void setLastSignIn(long lastSignIn) {
+        mLastSignIn = lastSignIn;
+    }
+
+    public boolean isStatus() {
+        return mStatus;
+    }
+
+    public void setStatus(boolean status) {
+        mStatus = status;
+    }
+
+    public User getUser(FirebaseUser firebaseUser) {
```

Comment:
e nên tạo 1 constructor cho user 
```suggestion
    public User(FirebaseUser firebaseUser) {
```